android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -785,6 +791,50 @@ fun ReadableMap.toLiveConfig(): LiveConfig = LiveConfig().apply {
     withDouble("minTimeshiftBufferDepth") { minTimeShiftBufferDepth = it }
 }
 
+fun ReadableMap.toHttpRequest(): HttpRequest? {
+    return HttpRequest(
+        getString("url") ?: return null,
+        getMap("headers")?.toMap(),
+        getString("body")?.toByteArrayFromBase64(),
+        getString("method") ?: return null,
+    )
+}
+
+private fun ByteArray.toBase64String(): String {
+    return Base64.encodeToString(this, Base64.NO_WRAP)
+}
```

Comment:
To add context here: I went for `Base64.NO_WRAP` as apparently `Base64.DEFAULT` adds a line break